### PR TITLE
Add cob4 23

### PR DESCRIPTION
**Pull request by HannesBachter**
needs https://github.com/ipa320/cob_navigation/pull/114
needs https://github.com/ipa320/cob_calibration_data/pull/163
needs https://github.com/ipa320/cob_supported_robots/pull/26

### CODE REVIEW

**Review by fmessmer — CHANGES_REQUESTED** [review]
PR for `cob_supported_robots` is missing

**Comment by HannesBachter** [comment]
PR for `cob_supported_robots` is here: https://github.com/ipa320/cob_supported_robots/pull/26

File: cob_moveit_config/robots/cob4-23/moveit/.setup_assistant [hunk 1]
```diff
@@ -0,0 +1,7 @@
+moveit_setup_assistant_config:
+  URDF:
+    package: cob_hardware_config
+    relative_path: robots/cob4-22/urdf/cob4-22.urdf.xacro
+    use_jade_xacro: true
+  SRDF:
+    relative_path: config/cob4-22.srdf
```
Comment (fmessmer) [inline cob_moveit_config/robots/cob4-23/moveit/.setup_assistant]: use cob4-23

File: cob_moveit_config/robots/cob4-23/moveit/config/cob4-23.srdf [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<robot name="cob4-2333">
```
Comment (fmessmer) [inline cob_moveit_config/robots/cob4-23/moveit/config/cob4-23.srdf]: use cob4-23

**Review by fmessmer — CHANGES_REQUESTED** [review]

**Review by fmessmer — APPROVED** [review]

**Pull request merged**